confAllChambers.py

```diff
@@ -52,6 +58,8 @@ def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,cName,ztrim):
 
     parser.add_option("--config", action="store_true", dest="config",
                       help="Set Configuration from simple txt files", metavar="config")
+    parser.add_option("--dictConfig", action="store_true", dest="dictConfig", default=False,
+                      help="Configure VFATs to custom chamber_default values", metavar="dictConfig")
```

Comment:
Q: 

Aren't the default values from `vfat_user_functions.py` l.15 that you linked applied every time when the `biasAllVFATs(ohboard,options.gtx,0x0,enable=False)` method is called at l.43 of `confChamber.py` ? If so, then shouldn't we just require:

```
if link in chamber_defaults.keys():
    Settings stored in chamber_defaults applied
    pass
```

as the alternative would be re-writing the defaults that have already been written at l.43 ? Unless, I'm misunderstanding what `biasAllVFATs` does, but it seems to me it's writing the register values with the `writeAllVFATs` method using the `parameters.defaultValues` values.